x-pack/plugins/lens/public/types.ts

```diff
@@ -701,6 +701,7 @@ export type VisualizationDimensionGroupConfig = SharedDimensionProps & {
   groupId: string;
   accessors: AccessorConfig[];
   supportsMoreColumns: boolean;
+  dimensionsTooMany?: number;
   /** If required, a warning will appear if accessors are empty */
   required?: boolean;
   requiredMinDimensionCount?: number;
```

Comment:
Follow-up: it looks like this one is no longer used anywhere after https://github.com/elastic/kibana/pull/139214